.devcontainer/devcontainer.json

```diff
@@ -21,13 +21,12 @@
 
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
   "forwardPorts": [
+    4222, // NATS Server 1
+    4223, // NATS Server 2
+    4224, // NATS Server 3
     5173, // CHEFS Frontend
     5432, // PostgreSQL
-    8080, // CHEFS Backend
-    8081,
```

Comment:
If someone is using the container for keycloak, for whatever reason, that would be port 8081.

Reply:
Local Keycloak has been removed so that port can go.